packages/common/http/src/request.ts

```diff
@@ -10,6 +10,21 @@ import {HttpContext} from './context';
 import {HttpHeaders} from './headers';
 import {HttpParams} from './params';
 
+/**
+ * Options to configure how TransferCache should be used to cache requests made via HttpClient.
+ *
+ * @param includeHeaders Specifies which headers should be included into cached responses. No
+ *     headers are included by default.
```

Comment:
(added docs for `includePostRequests` option)
```suggestion
 * @param includePostRequests Enables caching for POST requests. By default, only GET and HEAD requests are
 *     cached. This option can be enabled if POST requests are used to retrieve data (for example using GraphQL).
  * @param includeHeaders Specifies which headers should be included into cached responses. No
 *     headers are included by default.
```